apps/storybook-react-native/metro.config.js

```diff
@@ -1,5 +1,16 @@
 const { getDefaultConfig } = require('expo/metro-config');
 
-const config = getDefaultConfig(__dirname);
-
-module.exports = config;
+module.exports = (async () => {
+  const {
+    resolver: { assetExts, sourceExts },
+  } = await getDefaultConfig(__dirname);
+  return {
+    transformer: {
+      babelTransformerPath: require.resolve('react-native-svg-transformer'),
+    },
+    resolver: {
+      assetExts: assetExts.filter((ext) => ext !== 'svg'),
+      sourceExts: [...sourceExts, 'svg'],
+    },
+  };
+})();
```

Comment:
`react-native-svg-transformer` is required on the rendering side to render svgs properly. This is similar to the the setup in `metamask-mobile`